src/content/app/tools/blast/state/blast-api/blastApiSlice.ts

```diff
@@ -14,6 +14,9 @@
  * limitations under the License.
  */
 
+import { nanoid } from '@reduxjs/toolkit';
+import { type FetchBaseQueryError } from '@reduxjs/toolkit/query';
```

Comment:
Shouldn't this be `import type { FetchBaseQueryError }` since only one type is imported?

Reply:
✅